provisioning/AWS/aws-php-storage-driver-common-resources.yaml

```diff
@@ -73,6 +58,43 @@ Resources:
       ThumbprintList:
         - 6938fd4d98bab03faadb97b34396831e3780aea1
 
+  S3Bucket:
+    Type: "AWS::S3::Bucket"
+    Properties:
+      BucketName: ci-storage-driver-common
```

Comment:
Nemel byt pouze pro import/export lib dedikovany?

Reply:
Jo je to dedikovane chcel som to najskor pridat pre package `driver-common` ktory je jednoduhsi na testovanie, potom pokracovana import export